File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/DelicatessenOrderTracking.kt
package com.br.app.deliverydelicatessen.model

import java.util.Date

data class DelicatessenOrderTracking(
    var delicatessenOrderId: Int = 0,
    var statusOrderId: Int = 0,
    var followupDate: Date? = null,
    var statusOrder: StatusOrder? = null,
) {
}

File: financial/Models/DelicatessenOrderTracking.cs
using System.ComponentModel.DataAnnotations.Schema;
namespace Models
{
    [Table("DelicatessenOrderTracking")]
    public class DelicatessenOrderTracking : BaseEntity
    {
        public System.DateTime FollowupDate { get; set; }
        public int DelicatessenOrderId { get; set; }
        public int StatusOrderId { get; set; }

        [NotMapped]
        public virtual StatusOrder StatusOrder { get; set; }
        [NotMapped]
        public virtual DelicatessenOrder DelicatessenOrder { get; set; }

    }
}
